app/src/main/java/com/everest/movieapp/ui/adapters/SectionsPagerAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package com.everest.movieapp.ui.adapters
+
+import android.content.Context
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import com.everest.movieapp.ui.fragments.CurrentYearMovies
+import com.everest.movieapp.ui.fragments.PopularMovies
+
+/**
+ * A [FragmentPagerAdapter] that returns a fragment corresponding to
+ * one of the sections/tabs/pages.
+ */
+class SectionsPagerAdapter(
+    private val context: Context, fm: FragmentManager
+) :
+    FragmentPagerAdapter(fm) {
+
+    private var fragmentList: List<Fragment> = listOf(PopularMovies(), CurrentYearMovies())
+
+    override fun getItem(position: Int): Fragment {
+        // getItem is called to instantiate the fragment for the given page.
+        // Return a PlaceholderFragment (defined as a static inner class below).
+        return fragmentList[position]
+    }
+
+
+    override fun getPageTitle(position: Int): CharSequence {
+        val tabNames = ArrayList<String>()
+        tabNames.add("PopularMovies")
+        tabNames.add("CurrentYearMovies")
```

Comment:
you can initialise this list a class global level, No need to initialise every time